src/main/webapp/app/entities/participation/participation.model.ts

```diff
@@ -30,6 +32,8 @@ export abstract class Participation implements BaseEntity {
     public presentationScore: number;
     public results: Result[];
     public submissions: Submission[];
+    public student: User;
```

Comment:
This modeling is not correct, `student` and `exercise` are only part of a `student-participation`.

Reply:
It's probably just a remnant of the merge. I've removed it.